### PR TITLE
feat: pocket guide to mild evil skeleton

### PR DESCRIPTION
**Pull request by midgleyc**

### CODE REVIEW

File: src/data/items.txt [hunk 1]
```diff
@@ -11330,3 +11330,22 @@
...
+11314
+11315
+11316
+11317
+11318
+11319
+11320
+11321
+11322	Pocket Guide to Mild Evil	636540989	tinybook.gif	usable	t	0
+11323	Pocket Guide to Mild Evil (used)	508639228	tinybook.gif	usable		0
```
Comment (Veracity0) [inline src/data/items.txt:11351]: Surely this one is reusable.

File: src/data/items.txt [hunk 2]
```diff
@@ -11330,3 +11330,22 @@
...
+11313
+11314
+11315
+11316
+11317
+11318
+11319
+11320
+11321
+11322	Pocket Guide to Mild Evil	636540989	tinybook.gif	usable	t	0
```
Comment (Veracity0) [inline src/data/items.txt:11350]: Add these both to the switch in UseItemRequest.parseConsumption that calls UseItemRequest.itemToSkill - which looks up the modifier(.

**Review by Veracity0 — CHANGES_REQUESTED** [review]

File: src/net/sourceforge/kolmafia/request/FightRequest.java [hunk 3]
```diff
@@ -10249,6 +10249,18 @@ private static void payActionCost(final String responseText) {
...
         }
         break;
+
+      case SkillPool.PERPETRATE_MILD_EVIL:
+        if (
+        // Spookypocket
+        responseText.contains("so discombobulated")
+            ||
+            // no items
+            responseText.contains("even mildly evil")
```
Comment (gausie) [inline src/net/sourceforge/kolmafia/request/FightRequest.java:10259]: My understanding is that if there are no items missing it doesn't spend a charge

**Review by Veracity0 — APPROVED** [review]

File: src/net/sourceforge/kolmafia/request/FightRequest.java (hunk 3 above)
Comment (midgleyc) [reply src/net/sourceforge/kolmafia/request/FightRequest.java:10259]: I've tested and it definitely does spend a charge. The message does indicate that perhaps it shouldn't.

**Pull request merged**